src/shipit_code_coverage/tests/test_artifacts.py

```diff
@@ -59,3 +60,26 @@ def add_dir(files):
 
     with pytest.raises(Exception, message='suite and chunk can\'t both have a value'):
         a.get(chunk='xpcshell-7', suite='mochitest')
+
+
+MOCKED_PATH = [
+    'l', 'i', 'n', 'u', 'x', '_', 't', 'e', 's', 't', '/',
+    'c', 'o', 'd', 'e', '-', 'c', 'o', 'v', 'e', 'r', 'a', 'g',
+    'e', '-', 'g', 'r', 'c', 'o', 'v', '.', 'z', 'i', 'p', '_',
+    'c', 'o', 'd', 'e', '-', 'c', 'o', 'v', 'e', 'r', 'a', 'g',
+    'e', '-', 'g', 'r', 'c', 'o', 'v', '.', 'z', 'i', 'p',
+]
+
+
+@mock.patch('shipit_code_coverage.taskcluster.get_task_artifacts')
+@mock.patch('shipit_code_coverage.taskcluster.download_artifact')
+def test_download(mocked_download_artifact, mocked_get_task, LINUX_TEST_TASK_ARTIFACTS):
+    a = ArtifactsHandler([], [], [])
+    mocked_get_task.return_value = LINUX_TEST_TASK_ARTIFACTS['artifacts']
+    mocked_download_artifact.return_value = LINUX_TEST_TASK_ARTIFACTS['artifacts'][0]
+
+    a.download(LINUX_TEST_TASK_ARTIFACTS)
+
+    assert mocked_download_artifact.call_count == 1
+    assert mocked_get_task.call_count == 1
+    mocked_download_artifact.assert_called_with(MOCKED_PATH, 'MJIO3RWTRu2GhiE7_jILBw', 'public/code-coverage-grcov.zip')
```

Comment:
Can you use a string directly here instead of MOCKED_PATH?

Reply:
Yes, that was the way I was doing it, but it was causing a linting error. But I can move it back the right way so it won't give me any errors